src/test/rustdoc-js/filter-crate.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// should-fail
```

Comment:
This test seems odd to me. Wouldn't it be better to show that the result doesn't return anything with this filtering, rather than showing that the given results don't match what it returns?

Reply:
It's easier to look for something that isn't supposed to be there than the opposite. At least it sounds more logical to me.